packages/migrations/__tests__/migrations/5.37.0/002/ddb/002.test.ts

```diff
@@ -0,0 +1,214 @@
+import {
+    assertNotError,
+    createDdbMigrationHandler,
+    getPrimaryDynamoDbTable,
+    groupMigrations,
+    insertDynamoDbTestData as insertTestData,
+    logTestNameBeforeEachTest,
+    scanTable
+} from "~tests/utils";
+import { AcoFolders_5_37_0_002 } from "~/migrations/5.37.0/002/ddb";
+import { ACO_FOLDER_MODEL_ID, ROOT_FOLDER } from "~/migrations/5.37.0/002/constants";
+import { insertTestFolders } from "./insertTestFolders";
+import { FolderDdbWriteItem } from "../types";
+import { createLocalesData, createTenantsData } from "../common";
+
+jest.retryTimes(0);
+jest.setTimeout(900000);
+
+describe("5.37.0-002", () => {
+    const table = getPrimaryDynamoDbTable();
+
+    logTestNameBeforeEachTest();
+
+    it("should not run if no tenant found", async () => {
+        const handler = createDdbMigrationHandler({ table, migrations: [AcoFolders_5_37_0_002] });
+
+        const { data, error } = await handler();
+
+        assertNotError(error);
+        const grouped = groupMigrations(data.migrations);
+
+        expect(grouped.executed.length).toBe(0);
+        expect(grouped.skipped.length).toBe(1);
+        expect(grouped.notApplicable.length).toBe(0);
+    });
+
+    it("should not run if no locale found", async () => {
+        await insertTestData(table, [...createTenantsData()]);
+
+        const handler = createDdbMigrationHandler({ table, migrations: [AcoFolders_5_37_0_002] });
+
+        const { data, error } = await handler();
+
+        assertNotError(error);
+        const grouped = groupMigrations(data.migrations);
+
+        expect(grouped.executed.length).toBe(0);
+        expect(grouped.skipped.length).toBe(1);
+        expect(grouped.notApplicable.length).toBe(0);
+    });
+
+    it("should not run if no pages found", async () => {
```

Comment:
`pages` -> `folders` (but it doesn't really matter 😃)

I would also cover the case of an empty folder index, as we did [here](https://github.com/webiny/webiny-js/blob/next/packages/migrations/__tests__/migrations/5.36.0/001/ddb-es/001.test.ts#L219).